packages/components/psammead-media-player/src/Placeholder/index.jsx

```diff
@@ -28,18 +29,34 @@ const StyledPlayButton = styled(PlayButton)`
 `;
 
 const Placeholder = ({ onClick, service, src, srcset, mediaInfo }) => {
-  const { title, datetime, duration, durationSpoken, type } = mediaInfo;
+  const {
+    title,
+    datetime,
+    duration,
+    durationSpoken,
+    type,
+    guidance,
+  } = mediaInfo;
+  const renderPlaybutton = () => (
+    <StyledPlayButton
+      title={title}
+      service={service}
+      onClick={() => {}}
+      datetime={datetime}
+      duration={duration}
+      durationSpoken={durationSpoken}
+      type={type}
+    />
+  );
   return (
     <StyledPlaceholder onClick={onClick}>
-      <StyledPlayButton
-        title={title}
-        service={service}
-        onClick={() => {}}
-        datetime={datetime}
-        duration={duration}
-        durationSpoken={durationSpoken}
-        type={type}
-      />
+      {guidance ? (
+        <Guidance service={service} message={guidance}>
+          {renderPlaybutton()}
+        </Guidance>
+      ) : (
+        renderPlaybutton()
+      )}
```

Comment:
Have you considered stacking the elements here instead of combining them? This would prevent Play Button from being re-rendered every time Guidance's state changes.